EIPS/eip-2456.md

```diff
@@ -0,0 +1,166 @@
+should become eligible at 0600 UTC on a Wednesday, which would result in a transition during EMEA
+business hours.
+
+## Backwards Compatibility
+
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+
+### EIP-2124 - Fork identifier for chain compatibility checks
+
+EIP-2124 describes a fork identifier for chain compatibility checks, however it is defined in terms
+of future expected fork blocks. Since the exact block number is not known this format would need to
+be changed to accommodate the fork time. One alternative is to always communicate the
+`TRANSITION_TIME`. This will produce no conflicts with blocks until mainnet reaches 1.4 billion
+blocks (on the order of 700 years). Another alternative is to communicate the `TRANSITION_TIME`
+before the transition and then `TRANSITION_BLOCK` after transition. The downside to this alternative
+is that between the time of the transition and when clients upgrade their genesis files that
+unsynced nodes may not be able to find peers.
+
+### Fast Sync, Warp Sync, and Beam Sync
+
+There will be minor impacts on all of the non-archival synchronization mechanisms. Generally
+speaking when these protocols start executing the blocks they will need to know what the current
+upgrade is and will need to be able to detect it based on timestamps in the header.
+
+For Fast Sync (currently Geth, Besu, and Nethermind) the blocks are not executed until both the
+complete set of block headers and complete world state for the pivot point is downloaded. Clients
+will have all of the data on-hand to make the calculations. Up to two previous Transition Eligible
+Blocks will need to be examined.
+
+For Warp Sync (Parity) a similarly complete database is available before block execution starts.
+Similarly up to two previous Transition Eligible Blocks will need to be examined.
+
+For Beam Sync (Trinity) block execution begins prior to the complete history of block headers and
+downloading of world state finishes. In this case the two previous Transition Eligible Block headers
+will need to be downloaded and examined prior to the execution of the first block.
+
+For all of Fast, Warp, and Beam sync the historically noted transition block can be recorded in the
+genesis file as an aid to determining where the relevant transitions should occur, but the time
+transitions should still be verified. Hence it would not be advisable to simply switch to the
+historical node values for a time based transition.
+
+## Test Cases
+
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+
+> To be completed once Eligible for Inclusion
+
```

Comment:
```suggestion
```
This isn't currently a required suggestion, consider just leaving it out.

Reply:
```suggestion
## Test Cases
TBD

```
Alternatively, if you want to leave it in so you don't forget that is fine as well, just remove the comment.